common/src/main/java/whocraft/tardis_refined/common/capability/TardisLevelOperator.java

```diff
@@ -193,15 +193,15 @@ public boolean isTardisReady() {
 
     public boolean exitTardis(Entity entity, ServerLevel doorLevel, BlockPos doorPos, Direction doorDirection) {
 
-        if (!entity.level().isClientSide()){
+        if (!entity.level().isClientSide()) {
```

Comment:
This method provides a ``ServerLevel`` in the form of ``doorLevel`` which makes this check redundant as you would not have a ``ClientLevel`` on the server

Reply:
See comment for the entry